scripts/get

```diff
@@ -130,7 +130,12 @@ downloadFile() {
 # installs it.
 installFile() {
   HELM_TMP="/tmp/$PROJECT_NAME"
-  local sum=$(openssl sha -sha256 ${HELM_TMP_FILE} | awk '{print $2}')
+  local shac="sha1"
```

Comment:
- [ ] Consider renaming this variable something longer like "sha_cmd."

Reply:
Done